Problem running tutorial (npm test)

Using this version of npm:

```
npm --version
3.10.10
```

When I run `npm test`, at the point Selenium standalone gets fired up, it immediately dies with:

```
Starting selenium standalone server...
(node:11449) DeprecationWarning: util.puts is deprecated. Use console.log instead.
Selenium standalone server started at http://192.168.0.2:53052/wd/hub
/usr/local/lib/node_modules/protractor/node_modules/source-map-support/source-map-support.js:223
    return '\n    at ' + wrapCallSite(frame);
                       ^

TypeError: CallSite method toString expects CallSite as receiver


npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/nick/.nvm/versions/node/v7.3.0/bin/node" "/Users/nick/.nvm/versions/node/v7.3.0/bin/npm" "run" "protractor"
npm ERR! node v7.3.0
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! serenity-js-getting-started@1.0.0 protractor: `protractor ./protractor.conf.js --silent`
npm ERR! Exit status 7
npm ERR! 
npm ERR! Failed at the serenity-js-getting-started@1.0.0 protractor script 'protractor ./protractor.conf.js --silent'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the serenity-js-getting-started package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     protractor ./protractor.conf.js --silent
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs serenity-js-getting-started
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls serenity-js-getting-started
npm ERR! There is likely additional logging output above.


```

Am I suffering from RTFM without realising it?

--- a/package.json
+++ b/package.json
@@ -6,11 +6,11 @@
   "scripts": {
     "prepublish": "npm run setup",
     "preprotractor": "npm run webdriver:update -- --standalone",
-    "protractor": "protractor ./protractor.conf.js --silent",
+    "protractor": "./node_modules/protractor/bin/protractor ./protractor.conf.js --silent",
     "setup": "npm run typings -- install",
     "test": "npm run protractor",
     "typings": "typings",
-    "webdriver-manager": "webdriver-manager",
+    "webdriver-manager": "./node_modules/protractor/bin/webdriver-manager",
     "webdriver:update": "npm run webdriver-manager update"
   },
   "repository": {

